packer/ansible/roles/vault/tasks/main.yml

```diff
@@ -27,95 +38,120 @@
     system: yes
     group: vault
     createhome: no
-    home: /srv/vault/
+    home: "{{ vault_config_dir }}"
```

Comment:
Why are we setting the home directory to `vault_config_dir`?

Reply:
Hey @lingrino , 

It probably doesn't matter a ton but mainly because it is a real directory (/srv/vault does not actually exist).

I did do a cross check of the HashiCorp installation guide here, and in their guide they are also setting the service account home directory to the config directory: https://learn.hashicorp.com/vault/operations/ops-deployment-guide#step-2-install-vault